src/commands.ts

```diff
@@ -893,6 +939,10 @@ export function register(ctx: WorkspaceContext) {
         vscode.commands.registerCommand("swift.selectXcodeDeveloperDir", () =>
```

Comment:
I guess this and the referenced function can be removed now, as select toolchain does the equivalent and more

Reply:
Actually, I believe the DEVELOPER_DIR setting is still useful. There are cases where you would want to set it even though you've selected a public toolchain, for example.

Reply:
Possibly but I was using it as a method to select the toolchain from an Xcode install. I think including this and `select toolchain...` is confusing to the user.

Reply:
Fair enough. I've removed it, but a sufficiently advanced user can still use the configuration to set DEVELOPER_DIR if they need it.